### PR TITLE
391 display country column in projects

### PR DESCRIPTION
**Pull request by hkaczmarek90**
Added Column and Filter

### CODE REVIEW

**Comment by pniedzwiedzinski** [comment]
Isn't #397 the same?

**Comment by hkaczmarek90** [comment]
No.. it's almost the same.
#397 was in Drive model only column. This one is in Project, and here is filter added too

**Comment by arturtamborski** [comment]
Good work on that. Can you resolve the conflict and remove package-lock.json from PR?

**Comment by pniedzwiedzinski** [comment]
> Good work on that. Can you resolve the conflict and remove package-lock.json from PR?

I believe that what @arturtamborski meant was to remove the `package-lock.json` from the diff, not the whole file from the repository. This could help:

```
git checkout upstream/develop -- frontend/package-lock.json
```

File: backend/fleet_management/models.py [hunk 1]
```diff
@@ -57,6 +58,11 @@ def __str__(self):
         return self.title
 
 
+class ProjectAdmin(admin.ModelAdmin):
```
Comment (magul) [inline backend/fleet_management/models.py]: We tend to store derivatives of `ModelAdmin` in `admin.py`, not in `models.py`.

**Pull request merged**